compiler/version_check.go

```diff
@@ -1,9 +1,9 @@
-// +build go1.11
-// +build !go1.12
+// +build go1.12
+// +build !go1.13
 
 package compiler
 
-const ___GOPHERJS_REQUIRES_GO_VERSION_1_11___ = true
+const ___GOPHERJS_REQUIRES_GO_VERSION_1_12___ = true
 
 // Version is the GopherJS compiler version string.
-const Version = "1.11-2"
+const Version = "1.12-wip"
```

Comment:
When you're ready to merge the PR, push a commit that updates the version to `1.12-1` just before.